Source/Resources/en.lproj/Localizable.strings

```diff
@@ -19,13 +20,13 @@
 "billingDetails" = "Billing Details";
 "billingDetailsRequired" = "Billing Details*";
 "name" = "Name";
-"countryRegion" = "Country*";
-"addressLine1" = "Address line 1*";
+"countryRegion" = "Country";
+"addressLine1" = "Address line 1";
 "addressLine2" = "Address line 2";
-"state" = "County";
-"postalTown" = "Town*";
-"postcode" = "Postcode*";
-"phone" = "Phone*";
+"state" = "State";
```

Comment:
This may need a en_GB version. UK will expect County instead of State

Reply:
Good point @harry-brown-cko. Figma design has `State`.  @patrick-hoban-cko please advice about this.

This is from [Figma](https://www.figma.com/file/ABFqgxna0VH5TcfMgwnQHs/02-%E2%80%A2-Vision?node-id=552%3A4733): 
<img width="247" alt="Screenshot 2022-05-26 at 10 16 05 am" src="https://user-images.githubusercontent.com/102961713/170458005-01b128ae-80ec-4e77-af84-f965b6d8c707.png">